components/rhel/rhel-ose/Vagrantfile

```diff
@@ -64,6 +54,26 @@ Vagrant.configure(2) do |config|
     config.registration.password = ENV['SUB_PASSWORD']
   end
 
+  if ENV.has_key?('PROXY')
```

Comment:
Yes @praveenkumar . Well it should work and better to do it to be on safe side :smile: